### PR TITLE
Add HPOS specific k6 test suite

### PR DESCRIPTION
**Pull request by rrennick**
### Changes proposed in this Pull Request:

This PR modifies some of the `k6` merchant test functions to allow excluding individual tests. The use case here is excluding tests which are not relevant to HPOS performance testing. To that end, the PR adds `hpos-baseline-load.js` which excludes those tests.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. In the `plugins/woocommerce` folder, run `pnpm run env:dev --filter=woocommerce `
2. Run `pnpm run env:performance-init --filter=woocommerce`
3. In the `tests/performance` folder, run `k6 run tests/hpos-baseline-load.js`
4. Run `k6 run tests/wc-baseline-load.js`
5. Both test suites should run successfully.

### CODE REVIEW

**Review by alopezari — COMMENTED** [review]
I executed both the regular performance tests and the new HPOS ones and both ran successfully. I've found some wrong checks ([example](https://user-images.githubusercontent.com/45979455/236240615-d9353187-8c62-43ba-88a5-566c99e9cdde.png)) in both cases so we might want to review these, but I understand it's not something related to this PR so it shouldn't be blocked by this.

Also, I didn't find anything odd in the changes. Still, I would be more confident if @tammullen could review this too in case I missed anything.

Good job @rrennick and thank you for working on this! 🙌

**Review by tammullen — DISMISSED** [review]
Looks great, thanks for this!

**Comment by nigeljamesstevenson** [comment]
I also executed the tests without issue - this looks good -  thanks Ron!

File: plugins/woocommerce/tests/performance/requests/merchant/add-order.js [hunk 1]
```diff
@@ -170,75 +179,429 @@ export function addOrder() {
...
+				[ 'wc_order_action', '' ],
+				[ 'woocommerce_meta_nonce', `${ woocommerce_meta_nonce }` ],
+			] );
+
+			if ( cot_status === true ) {
+				admin_update_order_base = `${ admin_update_order_base }&id=${ hpos_post_id }`;
+				admin_update_order_params = cotOrderParams.toString();
+			} else {
+				admin_update_order_params = orderParams.toString();
 			}
```
Comment (jorgeatorres) [inline plugins/woocommerce/tests/performance/requests/merchant/add-order.js]: This part undoes the changes from #37719, which are already in `trunk`. Specifically, these lines:

https://github.com/woocommerce/woocommerce/blob/246053277e3f40cdbc06ce002fcaa4d66f504e68/plugins/woocommerce/tests/performance/requests/merchant/add-order.js#L378-L384

**Review by jorgeatorres — CHANGES_REQUESTED** [review]
Hi @rrennick,

Some of the changes in `merchant/add-order.js` override those I made in #37719 to prevent the problematic `&id=....&id=...&id=...` requests.

Could you please rebase so that we can keep those fixes in?

Other than that, this looks good to me. Thank you!

File: plugins/woocommerce/tests/performance/requests/merchant/add-order.js (hunk 1 above)
Comment (rrennick) [reply plugins/woocommerce/tests/performance/requests/merchant/add-order.js]: Nice catch @jorgeatorres . This should be addressed in https://github.com/woocommerce/woocommerce/pull/37665/commits/c0c260c42bbe1b80a4afb3b4aed91d41dd6c20e3

**Review by jorgeatorres — APPROVED** [review]
LGTM. Thank you @rrennick!

**Pull request merged**